src/main.rs

```diff
@@ -18,7 +18,7 @@ async fn main() -> Result<()> {
 		.filter_level(cli.verbose.log_level_filter())
 		.init();
 
-	let (public_ipv4, public_ipv6) = ip::get_ips().await?;
+	let (public_ipv4, public_ipv6) = ip::get_ips(&cli).await?;
```

Comment:
it's a total nit, but I'd probably just pass the bool instead of giving the "get_ips" function the entire cli args.